morty.go

```diff
@@ -1100,7 +1101,7 @@ func main() {
 		log.Println("Using IPv4 only direct connections.")
 	}
 
-	p := &Proxy{RequestTimeout: time.Duration(cfg.RequestTimeout) * time.Second,
+	p := &Proxy{RequestTimeout: time.Duration(cfg.RequestTimeout) * time.Minute,
```

Comment:
Do I understand correctly, that the timeout is increased to 600 minutes? It doesn't sound a good default for production usage.